cron-task/README.md

```diff
@@ -0,0 +1,26 @@
+# Running one-off tasks on cloud.gov
+
+This is an example of a one-off task that can be run on cloud.gov. More on how to do this can be [found in the docs](https://cloud.gov/docs/management/one-off-tasks/). Note, this example uses node.js, but you can use any language that has a cron-like library to do something similar.
+
+## Usage
+
+To run this app locally, use `npm start`. To deploy to cloud.gov, use the instructions below:
+
+```bash
+~$ cf push -f manifest.yml --health-check-type none --no-route
```

Comment:
The `health check` and `no route` can also be added in the manifest as well.  Not sure if this is worth pointing out.

Reply:
Ah, that's. good idea.